gravitee-apim-console-webui/src/scss/gio-global-style.scss

```diff
@@ -8,26 +8,42 @@
 
 $background: map.get(gio.$mat-theme, background);
 
-.gio-snack-bar-success {
-  white-space: pre-wrap;
-  background-color: mat.get-color-from-palette(gio.$mat-success-palette, default);
-  color: mat.get-color-from-palette(gio.$mat-success-palette, default-contrast);
-
-  .mat-button {
+.gio-snack-bar-success.mat-mdc-snack-bar-container {
+  .mdc-snackbar__surface {
```

Comment:
Is there a way to target the type of element (i.e. `div`, `a`) instead of using classnames? It would be helpful to avoid this same problem if the classes are ever renamed by MDC.

Reply:
It looks like referencing `simple-snack-bar` might work.